python/feathub/feature_views/derived_feature_view.py

```diff
@@ -39,8 +40,8 @@ class DerivedFeatureView(FeatureView):
     """
     Derives features by applying the given transformations on an existing table.
 
-    Supports per-row transformation, over window aggregation transformation and table
-    join.
+    Supports per-row transformation, over window aggregation transformation, table
+    join and Java UDF transformation.
```

Comment:
I think we need more discussion before exposing the Java UDF transform. Should we remove the docs in this PR? And we can support Java UDF transformation properly in a future PR.